nano/templates/shuttle_manipulator.tmpl

```diff
@@ -40,7 +40,7 @@ Title: Shuttle Manipulator UI
 {{else data.selectedMenuKey == 'stemp'}}
 	<div class="item">
 	{{for data.templates_tabs}}
-		{{:helper.link('select template', 'space-shuttle', {'select_template' : 1, 'shuttle_id' : value.shuttle_id}, (data.selected && data.selected.shuttle_id == value.shuttle_id) ? 'selected' : null)}}
+		{{:helper.link('select template', 'space-shuttle', {'select_template' : 1, 'shuttle_id' : value.shuttle_id}, (data.existing_shuttle.shuttle_id == value.shuttle_id) ? 'selected' : null)}}
```

Comment:
The text label of this button should be `value.shuttle_id`, instead of `select template`